src/utils/accounts.ts

```diff
@@ -18,6 +19,15 @@ export async function getAuthToken(): Promise<string | undefined> {
     }
 }
 
+export async function isLoggedIn(): Promise<boolean> {
+    try {
+        await getUser();
```

Comment:
is this working? `getUser()` will throw err if there is no user connected?

Reply:
Yes, it will throw `Error(GENEZIO_NOT_AUTH_ERROR_MSG)` if no token was found

Reply:
yes, but there is also the case when there is a token. but it is invalid

Reply:
I tested it, stop making guesses please